tests/memex/storage_test.py

```diff
@@ -218,6 +258,14 @@ def test_it_does_not_crash_if_no_text_or_tags(self, pyramid_request):
 
         storage.create_annotation(pyramid_request, data)
 
+    @pytest.fixture
+    def default_groupfinder(self, pyramid_config):
+        pyramid_config.registry[groups.GROUPFINDER_KEY] = groups.default_groupfinder
```

Comment:
Hmm, yeah, this is a bit odd. Redoing the contents of `memex.groups` `includeme` in the tests is probably something we'd like to avoid if possible, but I can't think of a simpler way right now.